tests/acceptance/30_custom_promise_types/07_python_library.cf

```diff
@@ -0,0 +1,82 @@
+######################################################
+#
+#  Basic test of promise module using the Python library
+#
+#####################################################
+body common control
+{
+    inputs => { "../default.cf.sub" };
+    bundlesequence  => { default("$(this.promise_filename)") };
+    version => "1.0";
+}
+
+#######################################################
+
+bundle common python_check
+{
+  classes:
+    "python_f_string_support" expression => returnszero("/usr/bin/python3 -c 'f\"string\"'", "useshell");
+}
+
+bundle agent init
+{
+  meta:
+      "test_skip_unsupported" string => "!python_f_string_support";
```

Comment:
So where does this test run? Is it even worth having?

Reply:
Yes, it runs on all 7 platforms marked as red in this testrun: https://ci.cfengine.com/job/testing-pr/13238/ (except for i386_linux_redhat_5 which failed for a different reason).